department-of-veterans-affairs/caseflow
# Updated hearing conversion to include emails and timezones for all types

Resolves [CASEFLOW-2103](https://vajira.max.gov/browse/CASEFLOW-2103)

### Description
Please explain the changes you made here.

### Acceptance Criteria
- [ ] Email fields can be edited when converting from a "Virtual" hearing

### Testing Plan
1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearings schedule: http://localhost:3000/hearings/schedule
2. Select one of the dockets from the list
3. Click the “Edit Hearing Details” link for one of the scheduled hearings
4. Select the second option from the “Hearing Type” dropdown
5. Optionally fill out the email and timezone fields
  - Optionally filll out the Judge dropdown (*Virtual only)
6. Click the convert hearing button
7. You should see a transition alert indicating that the emails are being sent followed by a success alert confirming the change


### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1706" alt="Screen Shot 2021-11-09 at 8 27 18 AM" src="https://user-images.githubusercontent.com/61989526/140964198-38c4b352-9a3b-4ae1-b2bd-24c8758221e6.png">|<img width="1699" alt="Screen Shot 2021-11-09 at 8 24 45 AM" src="https://user-images.githubusercontent.com/61989526/140964258-ff34edc3-1502-4c64-9655-77a92141bc29.png">
